src/components/Compare/ModalCompare.tsx

```diff
@@ -12,117 +12,124 @@ import Filters from 'components/Compare/Filters';
-        setShowFaqModal={props.setShowFaqModal}
-        createCompareShareId={props.createCompareShareId}
-        homepageScope={props.homepageScope}
-        setHomepageScope={props.setHomepageScope}
-        homepageSliderValue={props.homepageSliderValue}
-        setHomepageSliderValue={props.setHomepageSliderValue}
-        region={props.region}
-        vaccinesFirst={props.vaccinesFirst}
-        vulnerabilityFirst={props.vulnerabilityFirst}
-      />
-    </Fragment>
-  );
-};
+    return (
+      <Fragment>
+        <LockBodyScroll />
+        <ModalHeader isHomepage={props.isHomepage}>
+          {!disableFilters && (
+            <Filters
+              isHomepage={props.isHomepage}
+              countyTypeToView={props.countyTypeToView}
+              setCountyTypeToView={props.setCountyTypeToView}
+              stateId={props.stateId}
+              county={props.county}
+              geoScope={props.geoScope}
+              setGeoScope={props.setGeoScope}
+              isModal
+              sliderValue={props.sliderValue}
+              setSliderValue={props.setSliderValue}
+              homepageScope={props.homepageScope}
+              setHomepageScope={props.setHomepageScope}
+              homepageSliderValue={props.homepageSliderValue}
+              setHomepageSliderValue={props.setHomepageSliderValue}
+            />
+          )}
+          {/* Need explicit div to ensure close icon is on the right of the pop up */}
+          <div />
+          <CloseIcon onClick={() => props.handleCloseModal()} />
+        </ModalHeader>
+        <CompareTable
+          stateName={props.stateName}
+          county={props.county}
+          setShowModal={props.setShowModal}
+          isModal
+          isHomepage={props.isHomepage}
+          locations={props.locations}
+          currentCounty={props.currentCounty}
+          countyTypeToView={props.countyTypeToView}
+          setCountyTypeToView={props.setCountyTypeToView}
+          geoScope={props.geoScope}
+          setGeoScope={props.setGeoScope}
+          stateId={props.stateId}
+          sorter={props.sorter}
+          setSorter={props.setSorter}
+          sortDescending={props.sortDescending}
+          setSortDescending={props.setSortDescending}
+          sortByPopulation={props.sortByPopulation}
+          setSortByPopulation={props.setSortByPopulation}
+          sliderValue={props.sliderValue}
+          setSliderValue={props.setSliderValue}
+          setShowFaqModal={props.setShowFaqModal}
+          createCompareShareId={props.createCompareShareId}
+          homepageScope={props.homepageScope}
+          setHomepageScope={props.setHomepageScope}
+          homepageSliderValue={props.homepageSliderValue}
+          setHomepageSliderValue={props.setHomepageSliderValue}
+          region={props.region}
+          vaccinesFirst={props.vaccinesFirst}
+          vulnerabilityFirst={props.vulnerabilityFirst}
+        />
+      </Fragment>
+    );
+  },
+);
```

Comment:
Consider wrapping the component before exporting it instead to reduce nesting and the size of the diff

```tsx
export default React.forwardRef((props, ref) => 
  <ModalCompare {...props} />
);
```

Reply:
great idea, thanks.  why didn't I think of that?